src/devtools/client/debugger/src/components/TestInfo/TestSteps.tsx

```diff
@@ -1,17 +1,23 @@
 import React, { useContext, useMemo, useState } from "react";
 
 import { startPlayback } from "ui/actions/timeline";
+import { partialRequestsToCompleteSummaries } from "ui/components/NetworkMonitor/utils";
 import Icon from "ui/components/shared/Icon";
+import { getEvents, getRequests } from "ui/reducers/network";
 import {
   getReporterAnnotationsForTitle,
   getReporterAnnotationsForTitleEnd,
 } from "ui/reducers/reporter";
 import { useAppDispatch, useAppSelector } from "ui/setup/hooks";
-import { AnnotatedTestStep, Annotation, TestItem, TestStep } from "ui/types";
+import { AnnotatedTestStep, TestItem, TestStep } from "ui/types";
 
+import { NetworkEvent, getDisplayedEvents } from "./NetworkEvent";
 import { TestInfoContext } from "./TestInfo";
 import { TestStepItem } from "./TestStepItem";
 
+// const XHR_TYPE = "xhr";
+export const XHR_TYPE = "text/jsx";
```

Comment:
Is this intentional? seems like an odd value

Reply:
@jaril - This is my only real question. The other comments are mostly editorial and can be deferred.

Reply:
Bandaid for now since we seem to be getting incorrect types for network events. Kannan's on it, and when it's fixed we can just switch to the commented line.

Reply:
https://linear.app/replay/issue/RUN-867/chromium-netmonitor-fix-xhr-event-attributes